.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+## PR Description: *short_description*
+
+### Tickets:
+ - *ticket-id*
+
+### Details:
+ - *item1*
+ - *...*
+
+### Pull Request Readiness Checklist
+
+See details at https://github.com/openvinotoolkit/openvino/wiki/Contribute#making-a-good-pull-request
+
+- [ ] I agree to contribute to the project under Apache 2 License.
+- [ ] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenVINO
+- [ ] The PR is proposed to proper branch
+- [ ] There is reference to original bug report and related work
```

Comment:
What if it is a new feature?

Reply:
bug / feature can be selected via labels.

And sometimes it's hard to say - it's bug or feature :)

Reply:
I updated the message